website/docs/addon_maya_artist.md

```diff
@@ -297,11 +297,22 @@ Then I've put everything into `arm_rig` group.
 When you've prepared your hierarchy, it's time to create *Rig instance* in AYON.
 Select your whole rig hierarchy and go **AYON → Create...**. Select **Rig**.
 Set is created in your scene to mark rig parts for export. Notice that it has
-two products - `controls_SET` and `out_SET`. Put your controls into `controls_SET`
+four products - `controls_SET`, `out_SET`,  `skeletonMesh_SET` and `skeletonAnim_SET`. Put your controls into `controls_SET`
 and geometry to `out_SET`. You should end up with something like this:
 
 ![Maya - Rig Hierarchy Example](assets/maya/artist/rig_hierarchy_example.jpg)
 
+The other two products `skeletonMesh_SET` and `skeletonAnim_SET` are proposed for fbx workflow.
+
+The `skeletonMesh_SET` defines the static FBX skeleton and mesh, as an FBX product directly from the rig publish,
+you should put your skinned character geometry with skeleton in the set.
+
+The `skeletonAnim_SET` defines the animated content that should be published to FBX from an animation scene
+for the animation product type. You should include your animated skeleton data merely into the set.
```

Comment:
```suggestion
for the animation product type. This set should include only your animated skeleton.
```

Also, just wanted to state - I am still confused that `skeletonMesh` needs BOTH the mesh and the skeleton band this set only includes the skeleton. Why does the mesh one need both? It might be worth it if the docs would clarify the reasoning.